perfmetrics/scripts/ls_metrics/listing_benchmark.py

```diff
@@ -49,6 +52,8 @@
 )
 log = logging.getLogger()
 
+INSTANCE = fetch_metrics.INSTANCE
```

Comment:
Any reason to create new variable here?
Also, it seems it is unused. Same for PERIOD_SEC.

Reply:
Yes! It's unused. Thanks for pointing this out! I'll remove it